addons/mass_mailing/wizard/mailing_mailing_test_views.xml

```diff
@@ -20,6 +20,34 @@
             </field>
         </record>
 
+        <record id="mass_mailing_preview_view_form" model="ir.ui.view">
```

Comment:
I think we could merge the 2 forms into one as I suggested earlier and have a simple box with the emails and a button to send, similarly to what Luc suggested in the task pad.

(But please keep the custom JS widgets to a minimum for that as well, pure XML/css would be ideal).

Having 2 separate forms on the same wizard is very confusing.